test/room_test.rb

```diff
@@ -3,22 +3,45 @@
 require './lib/room'
 
 class RoomTest < Minitest::Test
+
+
   def test_it_exists
     room = Room.new(:bedroom, 10, '13')
     assert_instance_of Room, room
   end
 
   def test_it_has_a_category
-    skip
+
     room = Room.new(:bedroom, 10, '13')
     assert_equal :bedroom, room.category
   end
 
   def test_it_can_get_area
-    skip
+
     room1 = Room.new(:bedroom, 10, '13')
     room2 = Room.new(:living_room, 15, '12')
     assert_equal 130, room1.area
```

Comment:
This test is failing, because there is no `attr_reader` for `area`. You need to be running your tests and letting the errors guide implementation!